src/Smart.CFA.Pages/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
```

Comment:
I would move this section at the end of the file, where base is used, without naming it.